configure-places-in-the-sdk/places-monitor-extension/using-the-places-monitor-extension.md

```diff
@@ -44,7 +63,31 @@ import ACPPlacesMonitor
 
 {% tabs %}
 {% tab title="Android" %}
-This content is in progress.
+
+#### Java
+
+In your App's `OnCreate` method register the Places Monitor extensions:
+
+```java
+public class MobileApp extends Application {
+    @Override
+    public void onCreate() {
+        super.onCreate();
+        MobileCore.setApplication(this);
+        MobileCore.ConfigureWithAppId("yourAppId");
+        try {
+            PlacesMonitor.registerExtension(); //Register PlacesMonitor with Mobile Core
+            Places.registerExtension(); //Register Places with Mobile Core
+            MobileCore.start(null);
+        } catch (Exception e) {
+            //Log the exception
+         }
+    }
+}
+```
+
+**Important:** Places monitoring depends on the Places extension. When manually installing the Places Monitor extension, ensure that you also add the `places.aar` library to your project.
```

Comment:
In the 2nd sentence, reword the first part as follows:

"When you manually install the Places Monitor..."